src/js/common/schemaform/Address.jsx

```diff
@@ -171,6 +171,20 @@ class Address extends React.Component {
             formContext={formContext}
             onChange={(update) => {this.handleChange('street2', update);}}
             onBlur={onBlur}/>
+
+        {schema.properties.street3 ? <SchemaField
+            name="street3"
+            required={this.isRequired('street3')}
+            schema={schema.properties.street3}
+            uiSchema={uiSchema.street3}
```

Comment:
I'm not sure that it'll _really_ matter, but in `schemaform/definitions/address.js`, could we add a default `uiSchema.street3` like the other fields have? If nothing else, when we look at the definition file it'll act as an indication that we can, in fact, have 3 street lines if necessary.